package.json

```diff
@@ -30,11 +30,11 @@
     "karma-phantomjs-launcher": "^0.1.4",
     "es5-shim": "^3.3.0",
     "browserify": "^4.1.11",
-    "react": ">=0.11.1",
+    "react": ">= 0.12.1 < 1",
```

Comment:
My thinking was that anything greater than 0.* might break, but then since its under version one I guess even minor changes might break...